task_processing/tests/integration/mesos/steps/mesos_steps.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import re
+from subprocess import call
+from subprocess import PIPE
+from subprocess import Popen
+
+from pytest_bdd import given
+from pytest_bdd import then
+from pytest_bdd import when
+
+from task_processing.plugins.mesos.mesos_executor import MesosExecutor
+from task_processing.runners.sync import Sync
+
+
+@given('docker-compose build')
+def docker_compose_build():
+    call(['docker-compose', 'down'])
+    call(['docker-compose', 'pull'])
+    call(['docker-compose', '--verbose', 'build'])
+
+
+@given('mesos zookeeper')
+def mesos_zookeeper():
+    return Popen(
+        ['docker-compose', 'run', '--rm', 'zookeeper'],
+        stdout=PIPE,
+        stderr=PIPE
+    )
+
+
+@given('mesos zookeeper')
+def mesos_master():
+    return Popen(
```

Comment:
i thought about this but for now `Popen` is simple enough. eventually we'd just use `plumbum` as the bug referd to in `_run` docstring seems to have been fixed?